index.html

```diff
@@ -11,105 +11,105 @@
 </head>
 <body>
   <header>
```

Comment:
- [ ] [Optional] Please stick to the provided design as much as possible the two designs should be identical, consider using the same background blur to the headline section and use the same padding/margin and font size

|Your design|CC design|
|--|--|
|![Screenshot_20220526_193949](https://user-images.githubusercontent.com/25227022/170554954-5e55658e-2cf3-489e-a852-436e57a78bbd.png)|![Screenshot_20220526_194236](https://user-images.githubusercontent.com/25227022/170555165-bb8e68c4-fafc-43a1-ba87-679df3a8b762.png)|

Reply:
Done